frontend/pages/SoftwarePage/SoftwareOS/SoftwareOSTable/SoftwareOSTable.tests.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+import { createCustomRenderer } from "test/test-utils";
+
+import createMockUser from "__mocks__/userMock";
+import { createMockOSVersionsResponse } from "__mocks__/softwareMock";
+
+import SoftwareOSTable from "./SoftwareOSTable";
+
+// TODO: figure out how to mock the router properly.
+const mockRouter = {
+  push: jest.fn(),
+  replace: jest.fn(),
+  goBack: jest.fn(),
+  goForward: jest.fn(),
+  go: jest.fn(),
+  setRouteLeaveHook: jest.fn(),
+  isActive: jest.fn(),
+  createHref: jest.fn(),
+  createPath: jest.fn(),
+};
+
+describe("Software operating systems table", () => {
+  it("Renders the page-wide disabled state when software inventory is disabled", async () => {
+    const render = createCustomRenderer({
```

Comment:
`createCustomRenderer` is useful in situations where we want to specify something non-default about the testing context or the acting user and/or when we will want to `await` a user action such as clicking - see `AddHostsModal.tests.tsx` lines 41 and 50, for example.

In a situation like this where we don't need any of that, we can actually just use the `render` exported from react testing library. So line 2 would be `import { render, screen } from "@testing-library/react";`, and we can remove `import { createCustomRenderer } from "test/test-utils";` and it's callers altogether